hypha/apply/users/urls.py

```diff
@@ -174,7 +174,7 @@
     path(
         "account/two_factor/setup/complete/",
         RedirectView.as_view(url=reverse_lazy("users:backup_tokens"), permanent=False),
-        name="two_factor:setup_complete",
+        name="setup_complete",
```

Comment:
When I was looking at this issue. The `two_factor:setup_complete` was referenced from the hypha code so was assumed this namespace might be getting used by the library itself, so didn't decide to update it. 

If this works and don't causes any issue to the our workflow. This looks good to me.

Reply:
I checked the workflow, everything seems to be working fine for me.